docs-src/zdm-core/modules/migrate/pages/introduction.adoc

```diff
@@ -39,20 +39,27 @@ An important migration prerequisite is that you already have the matching schema
 
 == Migration phases
 
-First, a couple of key terms used throughout the {zdm-product} documentation and software components:
+First, a couple of key terms used throughout the ZDM documentation and software components:
 
 * **Origin:** This cluster is your existing Cassandra-based environment, whether it's open-source Apache Cassandra, DSE, or {astra_db} Classic.
 
 * **Target:** This cluster is the new environment to which you want to migrate client applications and data.
 
 For additional terms, see the xref:glossary.adoc[glossary].
 
+[TIP]
+====
+To get acquainted with the migration phases, begin by clicking the **Start** button below the diagram.
+You can optionally click the illustrated components to open a larger view. 
+Click again to return to the original view and use the navigation buttons. 
+====
```

Comment:
I think what you have works fine, but figured I'd take a stab at an alternative approach. I'm not sure it's better, but it's another option to think about in the future. There are also hypothetical SEO and site-search benefits to putting it under its own heading.

```suggestion
=== Migration interactive diagram

Click the *Start* button on the interactive diagram below to begin a walkthrough of the migration phases.
Click the components to open a larger view.
```